types/express-ejs-layouts/express-ejs-layouts-tests.ts

```diff
@@ -0,0 +1,7 @@
+import * as express from 'express';
+import * as expressEjsLayouts from 'express-ejs-layouts';
+
+function expressRequestHandlerTest() {
+    const app = express()
+        .use(expressEjsLayouts());
+}
```

Comment:
It needn't be put into a function, the test just checks, does not run.

Reply:
True. I prefer test case methods though - in other DefinitelyTyped submissions I had multiple test variations. Leaving the test methods in there help maintainers realize it's possible to do.